src/lib/components/tablev2/SingleSelectableContent.js

```diff
@@ -121,18 +131,12 @@ export default function SingleSelectableContent({
 
             return (
               <div {...cellProps} className="td">
-                {
-                  // if cell.column.cell is defined that means we have a custom cell renderer
-                  // otherwise it should be just a string and we can use the constrainted text component
-                  !cell.column.cell && typeof cell.value === 'string' ? (
-                    <ConstrainedText
-                      text={cell.value}
-                      tooltipPlacement={index === 0 ? 'bottom' : 'top'}
-                    />
-                  ) : (
-                    cell.render('Cell')
-                  )
-                }
+                {cell.column.Cell.name === 'defaultRenderer' &&
+                typeof cell.value === 'string' ? (
+                  <ConstrainedText text={cell.value} />
+                ) : (
+                  cell.render('Cell')
+                )}
```

Comment:
we need to change the constraintedText behavior .. if there is enough place, we should not have the tooltip displayed